src/value.rs

```diff
@@ -21,6 +22,9 @@ pub enum Value {
         range: Range<BigInt>,
         value: BigInt,
     },
+    Circuit(Vec<BigUint>),
+    CircuitModulus(BigUint),
+    CircuitOutputs(HashMap<u64, BigUint>),
```

Comment:
In Cairo Native the Outputs value it is represented as a vector rather than a HashMap. I think is better to use the same representation to make the trace dump easier to build (and visualize).